src/test/scala/mesosphere/marathon/MarathonTestHelper.scala

```diff
@@ -110,37 +112,26 @@ trait MarathonTestHelper {
     executor = "//cmd"
   )
 
-  def getSchemaMapper() = {
-    import com.fasterxml.jackson.annotation.JsonInclude
-    import com.fasterxml.jackson.module.scala.DefaultScalaModule
-    import mesosphere.jackson.CaseClassModule
-    import mesosphere.marathon.api.v2.json.MarathonModule
-
-    val schemaMapper = new ObjectMapper
-    schemaMapper.registerModule(DefaultScalaModule)
-    schemaMapper.registerModule(new MarathonModule)
-    schemaMapper.registerModule(CaseClassModule)
-    schemaMapper.setSerializationInclusion(JsonInclude.Include.NON_EMPTY)
-    schemaMapper
-  }
-  val schemaMapper = getSchemaMapper()
-
-  def getAppSchema() = {
+  lazy val appSchema = {
     val appJson = "/mesosphere/marathon/api/v2/AppDefinition.json"
     val appDefinition = JsonLoader.fromResource(appJson)
     val factory = JsonSchemaFactory.byDefault()
     factory.getJsonSchema(appDefinition)
   }
-  val appSchema = getAppSchema()
 
   def validateJsonSchema(app: V2AppDefinition, valid: Boolean = true) {
-    val appStr = schemaMapper.writeValueAsString(app)
+    import mesosphere.marathon.api.v2.json.Formats._
+    // TODO: Revalidate the decision to disallow null values in schema
+    // Possible resolution: Do not render null values in our formats by default anymore.
```

Comment:
Me, too. But this isn't part of this PR in my opinion.

I created https://github.com/mesosphere/marathon/issues/2081